packages/dart/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## [3.2.1](https://github.com/parse-community/Parse-SDK-Flutter/compare/dart-3.1.9...dart-3.2.1) (2023-01-14)
```

Comment:
Why is this jumping from `3.1.9` to `3.2.1`? If this is a fix it should to `3.1.10`

Reply:
I thought it should start from 1 after 9, that's why I changed it.

I changed it to version 3.1.10

Reply:
Got it. You may want to take a look at [semver](https://semver.org) which we are following in every Parse Platform repository.